### DISCUSSION TITLE
If i update from Jyers, do I need to reconfigure printer's settings from scratch?

### DISCUSSION THREAD

**Discussion by TheRadziu** [mriscoc/Ender3V2S1#978] [Q&A]
Currently I have Jyers with 3x3 probe on each print (G29). I know I'll most likely need to edit my start g-code to use UBL instead, but my question regards on printer settings such like XY CR touch offset, Z offset, esteps etc. Do they stay if I install this cfw over jyers' one? Or will i need to reconfigure the printer's settings from scratch?

**Comment by mriscoc** [comment]
Every time you change the firmware version, it is recommended to load the default settings and redo all the settings because the EEPROM structure might be different.
I recommend to do: https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files
